src/App.svelte

```diff
@@ -0,0 +1,494 @@
+<script>
+// import { onMount } from "svelte";
+let value = 0;
+let counter = 1;
+let themeValue = 1;
+let storage = [0,0];
+let operator = false;
+let operatorWhich = "";
+let firstTime = false;
+let dotClicked = false;
+
+const updateTheme = (a) => {
```

Comment:
You don't have to define this function here since it is called from only 1 place. You can use it inline from there.

Reply:
changing this is not worked for me